### DISCUSSION TITLE
Second instance

### DISCUSSION THREAD

**Discussion by AngelBarbero** [paukstelis/octoprint_deploy#121] [Q&A]
When I add the second instance and the printer detection finishes and it does not detect it (because I do not connect it), it does not create the instance, it sends me to the main menu.
Thanks for the help

**Comment by AngelBarbero** [comment]
Could the main instance file be cloned by giving it another name? If so, how could I get that copy to come out through port 5001?

**Comment by paukstelis** [comment]
Why not just connect the second printer since that is how it is supposed to work?

**Reply by AngelBarbero** [reply]
I am going to create 30 octoprints on 30 printers, each btt or pi board will have two printers connected. The idea is to create them in the office and then plug them directly into the ship.

**Comment by AngelBarbero** [comment]
I just tried connecting it to the printer and it doesn't detect it either. When I do the USB test if it detects it 🤔

**Reply by paukstelis** [reply]
If you are using the octoprint_deploy USB testing it is the exact same function that runs for testing as it does for the actual detection, so it can't be an issue there.

**Reply by AngelBarbero** [reply]
I got it, after trying many things in the end I touched the part of the code to skip that part, the problem may be with the board I am using. I just need to test that it works, thanks.
